CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-* Upgrade various dependencies.
+* Upgrade various dependencies [#393](https://github.com/clojure-emacs/refactor-nrepl/pull/393) & [#394](https://github.com/clojure-emacs/refactor-nrepl/pull/394).
```

Comment:
Just added the links back to the PRs.

The rest is just whitespace removal. I can revert all the changes below to CHANGELOG.md if you'd like, but right now I just want to submit this for review & go to sleep :)

Reply:
We've chatted about this, can be ignored, resolving